install_config/configuring_aws.adoc

```diff
@@ -218,3 +220,61 @@ $ systemctl restart origin-node
 ----
 endif::[]
 .  Add back any xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[labels on each node] that you previously had.
+
+[[aws-cluster-labeling]]
+== Labeling Clusters for AWS
+Starting with {product-title} version 3.7 of the installation program, if you
+configured AWS provider credentials, you must also ensure that all instances are
+labeled. This topic describes how to label an existing {product-title} cluster
+running on AWS.
+
+To correctly identify which resources are associated with a cluster, tag
+resources with the key `kubernetes.io/cluster/<name>,Value=<clusterid>`, where:
+
+* `<name>` is a unique name for the cluster.
+* `<clusterid>` is a cluster identifier unique to the AWS Availability Zone.
+
+Tagging all resources with the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+tag avoids potential issues with multiple zones or multiple clusters.
+
+[NOTE]
+====
+In versions prior to {product-title} version 3.6, this was
+`Key=KubernetesCluster,Value=clusterid`.
+====
+
+See xref:../architecture/core_concepts/pods_and_services.adoc#labels[Pods and
+Services] to learn more about labeling and tagging in {product-title}.
+
+[[aws-resources-that-need-tags]]
+=== Resources That Need Tags
```

Comment:
I can see this section was like this already, but i'm wondering if it can be merged into the 'Tagging an Existing Cluster' section below. This isn't adding much that the section below isn't already saying. WDYT?

Reply:
Thanks @bfallonf I am keeping this as it is for now because I think the next section is a procedure and this section is just calling what needs to be tagged. You are right that both of these sections should be combined, I will revisit this later.